templates/acm-certificate.yaml

```diff
@@ -24,6 +31,14 @@ Resources:
     Type: Custom::Certificate
     Properties:
       DomainName: !Sub '${SubDomain}.${DomainName}'
+      SubjectAlternativeNames:
+        Fn::If:
+          - CreateApexConfig
+          - - Ref: DomainName
+          - Ref: AWS::NoValue
+#      SubjectAlternativeNames:
+#      - !If [ CreateApexConfig, !Ref DomainName, !Ref 'AWS::NoValue' ]
+#      - !Ref DomainName
```

Comment:
Should these be removed prior to merging?

Reply:
This is intended.  The changes to the solution enable  the creation of an apex-domain in **addition** to a sub-domain.  The creation of an additional Alternative Name for the apex-domain to the certificate supports this.

Reply:
But they're commented out, so not used?

Reply:
Missed that .... commented out properties have been removed